crates/bdk-ldk/src/lib.rs

```diff
@@ -406,7 +407,18 @@ where
             ConfirmationTarget::HighPriority => 1,
         };
 
-        let estimate = self.client.estimate_fee(target_blocks).unwrap_or_default();
+        let estimate = match self.client.estimate_fee(target_blocks) {
+            Ok(estimate) => estimate,
+            Err(e) => {
+                tracing::warn!("Failed to get fee estimate. Using max sats per v byte {MAX_SATS_PER_V_BYTE} instead. Error: {e:?}");
+
+                // FIXME: Setting the fee rate to the maximum is a safe guard to prevent the channel
```

Comment:
I thought the problem of the force closure was because dour feetate was set to too high, compared to the counterpart's?

Reply:
I think it depends on the perspective..? I have to look into it to be completely sure.